jupyter_sphinx/ast.py

```diff
@@ -373,3 +414,52 @@ def get_widgets(notebook):
         # Don't catch KeyError, as it's a bug if 'widgets' does
         # not contain 'WIDGET_STATE_MIMETYPE'
         return None
+
+
+class CellOutputsToNodes(SphinxTransform):
+    """Use the builder context to transform a CellOutputNode into Sphinx nodes."""
+
+    default_priority = 700
+
+    def apply(self):
+        thebe_config = self.config.jupyter_sphinx_thebelab_config
+
+        for cell_node in self.document.traverse(JupyterCellNode):
+            output_bundle_node = _find_only_child_by_type(
+                cell_node, CellOutputBundleNode
+            )
```

Comment:
I propose to replace the function call with
```suggestion
            output_bundle_node, = cell_node.traverse(CellOutputBundleNode)
```
and similarly in the other two uses of the function `_find_only_child_by_type`.

The unpacking idiom is uncommon, and therefore has a chance to be confusing. But at the same time this implementation saves checking what the function actually does, and to an expert reader not familiar with the code base will be more understandable.